doc/theming.rst

```diff
@@ -117,6 +117,12 @@ These themes are:
   - **sidebarwidth** (an integer): Width of the sidebar in pixels.  (Do not
     include ``px`` in the value.)  Defaults to 230 pixels.
 
+  - **minbodywidth** (an integer): Minimal width of the document body in
+    pixels.  Defaults to 450 pixels.
+
+  - **minbodywidth** (an integer): Maximal width of the document body in
+    pixels.  Defaults to 800 pixels.
```

Comment:
Does this value also go well with themes not having sidebars (ex. haiku, scrolls and so on)?

Reply:
The refined version does now.